extension/modules/about-dialog.js

```diff
@@ -76,6 +76,7 @@ class AboutDialog {
             <img src="${chrome.runtime.getURL('images/AES-logo-128.png')}">
             <h2>AirlineSim Enhancement Suite</h2>
             <p>Version ${version}</p>
+            <p>Browser ${window.navigator.userAgent}<p>
```

Comment:
What’s your motivation for putting the userAgent string in the AES about screen?

Reply:
was for debug purposes if someone sends you a bug report on discord you have the AES version and exact browser version right there, in case there are browser-specific problems
if it bothers you, it can also be removed or moved elsewhere or hidden behind a "debug information" button or something